### ISSUE TITLE
YOLO V4 hyperparameter for achieveng higher mAP and accuracy after training

### ISSUE THREAD

**Issue by joan2212** [AlexeyAB/darknet#8841]
Hi everyone so i had this issue with setting the right hyperparameter such as batch size, subdivision, max batch, etc for achieveng higher mAP and accuracy after training with YOLO V4. I have 450 sample images and i keep getting 79% mAP after trying to training it for several time. Can anyone help me to set the right hyperparameter or cfg for getting at least 85% mAP?
